untraced/yksg/src/lib.rs

```diff
@@ -81,12 +87,12 @@ impl LocalMem {
     }
 
     /// Copy over the call arguments from another frame.
-    unsafe fn copy_args(&mut self, args: &[IRPlace], frame: &LocalMem) {
+    unsafe fn copy_args(&mut self, args: &[IRPlace], oframe: &Frame) {
```

Comment:
what is the `o` in `oframe` for?

Reply:
I'm just guessing but I think it might be `original frame`.

Reply:
"Other" or "Original" -- I just wanted something that isn't `frame` since that (mostly) means "the current frame".